modules/openapi-generator/src/main/resources/scala-akka-client/api.mustache

```diff
@@ -2,52 +2,51 @@
 package {{package}}
 
 {{#imports}}
-import {{import}}
+    import {{import}}
```

Comment:
@jaroslawZawila Did you add the spaces intentionally? what about line 7 to 9 ?

Reply:
I have removed the spaces.